backend/src/routes/food-trust-route.ts

```diff
@@ -0,0 +1,45 @@
+// import dependencies and initialize the express router
+import { Router } from "express";
+import { FoodTrustAPI } from "../integrations/food-trust/food-trust-api.service";
+
+const router = Router();
+
+const api = new FoodTrustAPI();
+
+router.get("/foodTrustAuthToken", async (req, res) => {
```

Comment:
We should probably delete the endpoint so someone can't just send a request and get our token.